### ISSUE TITLE
Error for .clean_columns with polars

### ISSUE THREAD

**Issue by TerryGamon** [aeturrell/skimpy#693]
in changelog I read:
v0.0.11
adding polars support

I am now encountering an error when using .clean_columns()
am I doing something wrong or is .clean_columns not supported for polars?
Thank you!

```
import polars as pl 
import skimpy #v0.0.14

df = pl.DataFrame({
    "Column Name 1": [1, 2, 3],
    "ANOTHER Column": [4, 5, 6],
    "More DATA": [7, 8, 9]
})
skimpy.clean_columns(df)
```
---> skimpy.clean_columns(df)
TypeCheckError: argument "df" (polars.dataframe.frame.DataFrame) is not an instance of pandas.core.frame.DataFrame

**Comment by aeturrell** [comment]
This is very much not the intended behaviour: it should work with polars too but I forgot to do it! 

Will add this when I can—thank you for flagging it.

**Comment by aeturrell** [comment]
Closed in #713
